.zenodo.json

```diff
@@ -0,0 +1,162 @@
+{
+  "title": "NetCDF Climate and Forecast (CF) Metadata Conventions",
+  "description": "Descriptive information: This document describes the CF conventions for climate and forecast metadata designed to promote the processing and sharing of files created with the netCDF Application Programmer Interface [NetCDF]. The conventions define metadata that provide a definitive description of what the data in each variable represents, and of the spatial and temporal properties of the data. This enables users of data from different sources to decide which quantities are comparable, and facilitates building applications with powerful extraction, regridding, and display capabilities.\nThe CF conventions generalize and extend the COARDS conventions [COARDS]. The extensions include metadata that provides a precise definition of each variable via specification of a standard name, describes the vertical locations corresponding to dimensionless vertical coordinate values, and provides the spatial coordinates of non-rectilinear gridded data. Since climate and forecast data are often not simply representative of points in space/time, other extensions provide for the description of coordinate intervals, multidimensional cells and climatological time coordinates, and indicate how a data value is representative of an interval or cell. This standard also relaxes the COARDS constraints on dimension order and specifies methods for reducing the size of datasets.",
```

Comment:
We should drop the reference links - "[NetCDF]" and "[COARDS]" from the description text. Also, do we need the "Descriptive information:" at the beginning? It seems redundant.

Reply:
Could I suggest changing 

> The extensions include metadata that provides a precise definition of each variable via specification of a standard name, describes the vertical locations corresponding to dimensionless vertical coordinate values, and provides the spatial coordinates of non-rectilinear gridded data.

to 

"The extensions include metadata that provide a precise definition of each variable via specification of a standard name, describe the vertical locations corresponding to dimensionless vertical coordinate values, and provide the spatial coordinates of non-rectilinear gridded data, as well as data on unstructured meshes."

Also - just checking that the `\n` is intentional!

Reply:
Dear Gui

Thanks for working on this. I commented on this PR in the associated [issue](https://github.com/cf-convention/cf-conventions/issues/127#issuecomment-1921274631). Please could you look at that? In particular, I think that David's suggestion, while reasonable, should _not_ be made in this PR, because (if I understand correctly) the text you have used is the abstract of the conventions document.

Jonathan

Reply:
Shall I mark this 'conversation' as resolved?

Reply:
Works for me.